File: championship-manager-frontend/src/app/entities/group-information.ts
import { Team } from "./team";

export interface GroupInformation{
    id: number;
    first_place: Team;
    second_place: Team;
    third_place: Team;
    fourth_place: Team;
    first_place_points: number;
    second_place_points: number;
    third_place_points: number;
    fourth_place_points: number;
}

File: championship.manager/src/main/java/api/championship/manager/models/GroupInformation.java
package api.championship.manager.models;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Table(name = "tb_group_information")
@Getter
@Setter
public class GroupInformation {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @ManyToOne
    private Team first_place;
    @ManyToOne
    private Team second_place;
    @ManyToOne
    private Team third_place;
    @ManyToOne
    private Team fourth_place;
    private int first_place_points;
    private int second_place_points;
    private int third_place_points;
    private int fourth_place_points;
    private LocalDateTime deletionDate;

    public GroupInformation(){}
}
